### PR TITLE
Feature: Equipment section

### PR DESCRIPTION
**Pull request by brbrakus**

### CODE REVIEW

File: styles/main.scss [hunk 1]
```diff
@@ -23,6 +25,8 @@ body {
 
 h1 {
   font-size: 40px;
+  text-shadow: 0px 1px 2px rgba(0, 0, 0, 0.2);
```
Comment (MiroDojkic) [inline styles/main.scss]: This color is used multiple times for text-shadow, could you please refactor to variable?

File: styles/header.scss [hunk 2]
```diff
@@ -49,9 +55,11 @@ header {
     margin-right: 7%;
     margin-right: 7vw;
 
+    text-transform: uppercase;
+    font-size: 90%;
```
Comment (MiroDojkic) [inline styles/header.scss]: Are we sure we want font size to scale with parent container?

File: styles/header.scss [hunk 3]
```diff
@@ -34,9 +39,10 @@ header {
         margin-left: 6%;
         margin-left: 2vw;
 
-        font-size: 26px;
+        font-size: 22px;
         letter-spacing: 0.05em;
         font-weight: 600;
+        text-shadow: 0px 0px 1px rgba(0, 0, 0, 0.2);
```
Comment (MiroDojkic) [inline styles/header.scss]: Refactor color to variable

File: styles/equipmentSection.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  flex-flow: column nowrap;
+
+  padding-top: 100px;
+  padding-bottom: 100px;
+  align-items: center;
+
+  background-color: #2980B9;
+
+    .equipment-header {
+    display: flex;
```
Comment (MiroDojkic) [inline styles/equipmentSection.scss]: Should be indented

File: styles/equipmentSection.scss [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  width: 100vw;
+  flex-flow: column nowrap;
+
+  padding-top: 100px;
+  padding-bottom: 100px;
+  align-items: center;
+
+  background-color: #2980B9;
+
+    .equipment-header {
```
Comment (MiroDojkic) [inline styles/equipmentSection.scss]: Not aligned with the props

File: styles/equipmentSection.scss [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    width: 100%;
+    width: 100vw;
+    flex-flow: column nowrap;
+    margin-bottom: 100px;
+
+    h1 {
+      color: white;
+      text-align: center;
+    }
+  }
```
Comment (MiroDojkic) [inline styles/equipmentSection.scss:24]: Think there might be missing one curly bracket here 🤔

File: styles/equipmentSection.scss [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+    flex-flow: row wrap;
+
+    margin-bottom: 30px;
+    justify-content: space-around;
+    text-align: left;
+
+    .equipment-card {
+      display: flex;
+      flex-flow: column nowrap;
+      background-size: contain;
```
Comment (MiroDojkic) [inline styles/equipmentSection.scss:39]: It's good this is explicit but eventually we should use shorthand for background

File: styles/equipmentSection.scss [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+      width: 310px;
+      height: 430px;
+
+      color: $link-color;
+
+      padding: 20px;
+      border-radius: 10px;
+      box-shadow: 2px 1px 15px rgba(0, 0, 0, 0.8);
+
+      h3{
```
Comment (MiroDojkic) [inline styles/equipmentSection.scss]: Space between bracket and selector. Might be good to use some formatter in editor (i.e. prettier)

File: styles/equipmentSection.scss (hunk 6 above)
Comment (brbrakus) [reply styles/equipmentSection.scss:24]: All brackets are closed :)

**Review by MiroDojkic — APPROVED** [review]

**Pull request merged**